core/keyserver/epochs.go

```diff
@@ -36,27 +36,55 @@ var (
 	maxPageSize = int32(2048) // 8MB
 )
 
-// GetEpoch returns a list of mutations paged by epoch number.
+// GetLatestEpoch returns the latest epoch. The current epoch tracks the SignedLogRoot.
+func (s *Server) GetLatestEpoch(ctx context.Context, in *pb.GetLatestEpochRequest) (*pb.Epoch, error) {
+	// Lookup log and map info.
+	d, err := s.domains.Read(ctx, in.DomainId, false)
+	if err != nil {
+		glog.Errorf("adminstorage.Read(%v): %v", in.DomainId, err)
```

Comment:
Might want to prefix the errors with the method. Up to you.

Reply:
done